File: work/sender/src/com/sender/send/bigdog.java
package com.sender.send;

import com.sender.impl.mailSenderTask;
import org.apache.log4j.Logger;

/**
 * Created with IntelliJ IDEA.
 * User: bigdog
 * Date: 10/18/12
 * Time: 6:45 PM
 * To change this template use File | Settings | File Templates.
 */
public class bigdog {
    private static Logger logger = Logger.getLogger(bigdog.class);

    public void work() {
        new Runnable() {
            @Override
            public void run() {
                new Sender().send(new mailSenderTask().getTaskbean());
//                new mailSenderTask().getTaskbean();
            }
        }.run();
    }


    public static void main(String[] ss) {
        new bigdog().work();
    }
}

File: work/sender/web/js/bigdog.js
/**
 * Created with IntelliJ IDEA.
 * User: bigdog
 * Date: 10/26/12
 * Time: 8:07 PM
 * To change this template use File | Settings | File Templates.
 */
 function changework(){
    $.post("index.jsp", {swicth:'2'}, function
        (databk) {
        databk = databk.trim();
        alert(databk);

    });

}
 function refresh() {
     $.post("index.jsp", {swicth:'1'}, function
         (databk) {
         databk = databk.trim();
         $("#total_now").html(databk);
     });
 }